OrdersApplication/OrdersViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  ViewController.swift
+//  OrdersApplication
+//
+//  Created by Esraa Khaled   on 22/09/2022.
+//
+
+import UIKit
+
+class OrdersViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!{
+        didSet{
+            tableView.delegate = self
+            tableView.dataSource = self
+        }
+    }
+    @IBOutlet weak var previous: UIButton!
+    @IBOutlet weak var current: UIButton!
+    var section1Expanded = false
+    var section2Expanded = false
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureTableView()
+        setupButtonAppearance()       
+    }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.navigationBar.barStyle = UIBarStyle.black
+        self.navigationController?.navigationBar.tintColor = UIColor.white
+        let appearance = UINavigationBarAppearance()
+        appearance.backgroundColor = .brandPrimary
+        appearance.titleTextAttributes = [.foregroundColor: UIColor.white]
+        appearance.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
+        navigationController?.navigationBar.tintColor = .white
+        navigationController?.navigationBar.standardAppearance = appearance
+        navigationController?.navigationBar.compactAppearance = appearance
+        navigationController?.navigationBar.scrollEdgeAppearance = appearance
```

Comment:
Add oncifugrations should be in `viewDidLoad`

Reply:
solved